x-pack/plugins/enterprise_search/public/applications/app_search/components/crawler/components/add_domain/add_domain_form_submit_button.tsx

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { useActions, useValues } from 'kea';
+
+import { EuiButton } from '@elastic/eui';
+
+import { i18n } from '@kbn/i18n';
+
+import { AddDomainLogic } from './add_domain_logic';
+
+export const AddDomainFormSubmitButton: React.FC = () => {
+  const { submitNewDomain } = useActions(AddDomainLogic);
+
+  const { hasValidationCompleted } = useValues(AddDomainLogic);
+
+  return (
+    <EuiButton fill type="button" disabled={!hasValidationCompleted} onClick={submitNewDomain}>
+      {i18n.translate('xpack.enterpriseSearch.appSearch.crawler.addDomainForm.submitButtonLabel', {
+        defaultMessage: 'Add Domain',
```

Comment:
```suggestion
        defaultMessage: 'Add domain',
```

I think lowers case is preferred in Kibana

Reply:
I've updated this in ad04a0b